src/charm.py

```diff
@@ -263,5 +263,20 @@ def ip_is_valid(ip_address: str) -> bool:
         return False
 
 
+def ip_in_cidr_format_is_valid(ip_address: str) -> bool:
+    """Check whether given IP config is in CIDR format and valid.
+
+    Args:
+        ip_address (str): IP address in CIDR format
+
+    Returns:
+        bool: True if given IP address is valid
+    """
+    if "/" not in ip_address:
+        logger.warning(f"The IP address: {ip_address} is expected in CIDR format.")
```

Comment:
Please avoid string formatting in logging. Reference [here](https://docs.google.com/document/d/1H5W2oi7cuTaz86taTQiITv49jpJNKzzDZ2G25IQpE4w/edit#heading=h.r5obiqmhwoy3)

Reply:
Yes, I was aware of security risk but somehow I forgot while implementing that case.
Done